src/ui/fx-control.js

```diff
@@ -17,6 +18,8 @@ class FxControl extends XfAbstractControl {
     super();
     this.inited = false;
     this.attachShadow({ mode: 'open' });
+
+    this.subForm = null;
```

Comment:
ok, this early in the process but just as a reminder i'd like to avoid the term 'form' altogether as it's still misleading - a sub component could well just be a model doing something (like converting, filtering) and not even have a visual representation. I suggest 'subcontrol'.

Reply:
in a next step this would become an array right? We need to start start somewhere...

Reply:
> in a next step this would become an array right? We need to start start somewhere...
    How do you mean? Multiple subforms?

Yes, multiple instances of the same subform should be possible to make it really useful. See example in Spec https://www.w3.org/community/xformsusers/wiki/XForms_2.0#The_Control_Element